### PR TITLE
clarify what happens with version in SDP o= lines during roll back

### PR DESCRIPTION
**Pull request by fluffy**
Fixes https://github.com/rtcweb-wg/jsep/issues/239

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -2308,11 +2308,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
                     <list style="symbols">
 
-                        <t>The fields of the "o=" line MUST stay the
-                        same except for the &lt;session-version&gt;
-                        field, which MUST increment if the session
-                        description changes in any way, including the
-                        addition of ICE candidates.</t>
+                        <t>The fields of the "o=" line MUST stay the same except
+                        for the &lt;session-version&gt; field, which MUST
+                        increment to be greater than any other SDP offers in
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: s/increment to/

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -2308,11 +2308,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                        this session if the session description changes in any
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: s/changes/has changed/

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -2308,11 +2308,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                        way, including the addition of ICE candidates. This
+                        means that if an Offer with version 12 was generated,
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: We seem to be using lower-case offer.

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -2308,11 +2308,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                        then an Offer with version with 13 was generated, then
+                        the SDP was rolled back to the Offer with version 12,
+                        the next offer would have a version 14 or larger. </t>
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Can you rewrite this with more reference to the APIs? I.e., "If createOffer() returns a ..."

**Comment by juberti** [comment]
The idea here seems right, but I don't think this is the right place in the overall text for it. Specifically, we need to point out that this affects even getLocalDescription, so we probably need to cover this in section 3.

**Comment by fluffy** [comment]
Replaced by https://github.com/rtcweb-wg/jsep/pull/277

**Pull request closed without merging**